go/kbnm/findbin_nix.go

```diff
@@ -0,0 +1,14 @@
+package main
+
+// +build !windows
+
+import (
+	"path/filepath"
+)
+
+const keybaseBinary = "keybase"
+
+// guessKeybasePath makes a platform-specific guess to where the binary might be.
```

Comment:
Maybe clarify that this only gets triggered in the case where the `kbnm` binary isn't adjacent to the `keybase` binary?